### PR TITLE
feat: update app description

### LINKED ISSUE #790: 🚀 Feature: update app description
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

Currently, the website description is "Find your npm packages eligible for Tidelift funding 💸." I find this a bit difficult to understand, so I suggest changing it to "Check if your npm packages are eligible for Tidelift funding 💸" so it's more straightforward.

### Additional Info

I've also noticed that the same description appears in other places in the app, such as metadata, `README.md`, and `package.json`, so those will likely need updating as well to maintain consistency.

### PR DESCRIPTION
**Pull request by maggienegm**
## PR Checklist

- [x] Addresses an existing open issue: fixes #790 
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

The app description across the app has been updated from "Find your npm packages eligible for Tidelift funding 💸" to "Check if your npm packages are eligible for Tidelift funding 💸".

The following locations were updated: layout (i.e. home page), metadata, `README.md`, and `package.json`.

✨

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import "./globals.css";
 const raleway = Raleway({ subsets: ["latin"] });
 
 export const metadata = {
-	description: "Find your npm packages eligible for Tidelift funding. 💸",
+	description:
```
Comment (maggienegm) [inline src/app/layout.tsx:8]: I just want to note that the prettier check doesn't pass if I have the description property all on one line, as in:

````js
export const metadata = {
	description: "Check if your npm packages are eligible for Tidelift funding. 💸<",
	title: "Tidelift Me Up",
};
````

This is why the description property code is written/formatted this way.

Comment (JoshuaKGoldberg) [reply src/app/layout.tsx:8]: Yip yip, this is expected and 👍

**Review by JoshuaKGoldberg — APPROVED** [review]
Perfect, thanks for the thoroughness in changes! 💯

**Comment by JoshuaKGoldberg** [comment]
@all-contributors please add @maggienegm for code.

> 🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).
> Not all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.
> ...and of course, thank you for contributing! 💙

**Pull request merged**